protocol/transport/transport.go

```diff
@@ -258,6 +258,8 @@ func (t *Transport) RetrieveRawAll() (map[Filter][]*types.Message, error) {
 		}
 
 		for i := range msgs {
+			// TODO: find the filter for the msg based on chatID in the message and map it properly. or better this is done in filter layer itself?
+			// something like t.FilterByChatID()
```

Comment:
Ah, i am not sure if this is needed.
Will wait for review from status team to understand the impact and then either remove this or track it separately.

Reply:
I was little doubtful as to how this will have an impact on any other flows. 
I had noticed that messages received by a filter can be fetched using `chatID`. But the actual `chatID` of the message will be different than the one stored in filter (i.e  `<communityID>-memberUpdatesChannelID`). 

I was wondering if message retrieval flow will get affecting in some form if i don't use the actual chatID of the message. The caller of RetrieveAll has to retrieve messages using  `<communityID>-memberUpdatesChannelID` as chatID and will not have flexibility to filter messages based on actual chatID. 
Because the result of this function will now index all messages with single filter rather than chatID specific filters.